src/support/project.ts

```diff
@@ -42,9 +42,17 @@ export const basePath = (srcPath = ""): string => {
     return path.join(config<string>("basePath", ""), pathForPhpEnv(srcPath));
 };
 
-export const projectPath = (srcPath = ""): string => {
+export const projectPath = (
+    srcPath = "",
+    workspaceFolder?: string | undefined,
+): string => {
     srcPath = basePath(srcPath);
 
+    // Support for multi-root workspaces
+    if (workspaceFolder) {
+        return path.join(workspaceFolder, srcPath);
+    }
+
```

Comment:
I think multi-root workspaces is already covered by the for-loop in this function, so passing in `workspaceFolder` to this function should not be needed.

Is there something I am missing @N1ebieski ?

Reply:
I agree with you. The existing implementation fails when `srcPath` exists in multiple workspaces and your approach handles that perfectly!

You can mark this comment as resolved, too.